script/elevate-cpanel.PL

```diff
@@ -1193,13 +1194,7 @@ sub run_stage_4 ($self) {
 
     $self->post_leapp_update_restore();
 
-    my @known_modules_that_dont_convert = qw{libtermkey msgpack btrfs-progs elevate-release
-      leapp leapp-data-almalinux leapp-data-rocky leapp-data-cloudlinux leapp-upgrade-el7toel8
-      python2-leapp};
-    my @to_remove = grep { Cpanel::Pkgr::is_installed($_) } @known_modules_that_dont_convert;
-
-    INFO("Removing known cruft that various 3rdparties leave behind. Also removing leapp.");
-    $self->ssystem( qw{/usr/bin/dnf -y erase}, @to_remove );
+    $self->run_component_once( 'UnconvertedModules' => 'post_leapp' );
```

Comment:
I considered it out of scope for this case.  We should address the component logic via RE-253 and RE-254.  At a glance, I think Re-254 should happen first so that we can work on getting most everything that needs to be a component in a component before we address the ordering.